src/library-authoring/routes.ts

```diff
@@ -82,13 +82,7 @@ export const useLibraryRoutes = (): LibraryRoutesData => {
       route = ROUTES.HOME;
     } else if (insideCollections) {
       // We're inside the Collections tab,
-      route = (
-        (collectionId && collectionId === params.collectionId)
-          // now open the previously-selected collection,
-          ? ROUTES.COLLECTION
```

Comment:
This was causing a redirect to the Collection page on a second click on the collection card.